torchbenchmark/operators/FusedLinearCrossEntropy/operator.py

```diff
@@ -0,0 +1,98 @@
+import argparse
+from typing import List, Optional, Generator, Callable
+import torch
+from torchbenchmark.util.triton_op import (
+    BenchmarkOperator,
+    register_benchmark,
+)
+from liger_kernel.transformers.fused_linear_cross_entropy import LigerFusedLinearCrossEntropyLoss
```

Comment:
We probably also need to setup how to install linger_kernel (e.g., adding it as a submodule, or install it in https://github.com/pytorch/benchmark/blob/main/userbenchmark/triton/install.py)

Reply:
add the installation for liger-kernel in 
 https://github.com/pytorch/benchmark/pull/2485/commits/9580b0f964b29f0599bbd872840a4540370e7234

Reply:
Bumping transformer version in https://github.com/pytorch/benchmark/pull/2488

Reply:
The transformer version has been updated, can you help rebase this PR on the trunk?

Reply:
done